src/scripts/app.ts

```diff
@@ -1287,7 +1287,7 @@ export class ComfyApp {
 
         // Store all widget values
         if (widgets) {
-          for (const i in widgets) {
+          for (let i = 0; i < widgets.length; i++) {
```

Comment:
There are existing widgets length calls to in litegraph that works fine with all existing widgets hacks. https://github.com/Comfy-Org/litegraph.js/blob/e87b17fb276083380f574ce5f30b7ffad7c4999c/src/LGraphNode.ts#L681-L689 so this conversion should be fine.